src/sass/_base.scss

```
@@ -0,0 +1,146 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    color: $color-text-body;
+    font-family: "Commissioner", sans-serif;
+    font-weight: 400;
+    margin: 0;
+    height: 100vh;
+}
+
+.sidenav {
+  background-color: $color-grey;
+  position: fixed;
+  height: 100%;
+  width: 432px;
+  top: 0;
+  left: 0;
+  color: $color-offwhite;
+  padding: 15px 35px;
+  
+  ul {
+    padding: 0;
+  }
+  
+  li {
+    list-style-type: none;
+    text-align: center;
+    margin: 15% 0;
+    
+    a {
+      color: $color-offwhite;
+      text-decoration: none;
+      font-size: 2em;
+      font-weight: 100;      
+    }  
+  }
+  
+  .social { 
+    img {
+      width: 32%;
+    }
+  }
+}
+
+main {
+    flex: 1 0 auto;
+    padding-left: 432px;
+    background-image: url("https://cdn.glitch.com/779b971d-9c2d-457f-ac8f-945d722901a7%2Fbs-img1.jpg?v=1605829317131");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+  
+  section {
+    padding: 25px;
+    background-color: $color-offblack;
+    z-index: 1;
+    margin-right: 30%;
+    height: 100vh;
+    overflow-y: auto;
+  }
+  
+  section::-webkit-scrollbar-thumb {
+    background: #000;
+  }
+}
+
+h2 {
+    color: $color-orange;
+    margin: 0;
+    opacity: 1;
+}
+
+p {
+    color: $color-offwhite;
+    opacity: 1;
+    padding: 0 30px;
+}
+
+fieldset {
+  text-area {
+    color: $color-grey;
+  }
+  label {
+    color: $color-offwhite;
+  }
+  legend {
+    color: $color-offwhite;
+  }
```

Consider combining these, if all things are the same:
```suggestion
  label,
  legend {
    color: $color-offwhite;
  }
```